src/ddspdrum/module.py

```diff
@@ -353,23 +396,41 @@ def __init__(
         sustain_duration: float,
         pitch_adsr: ADSR = ADSR(),
         amp_adsr: ADSR = ADSR(),
-        vco: VCO = VCO(),
+        vco_1: VCO = SineVCO(),
+        vco_2: VCO = SquareSawVCO(),
+        mixer: Mixer = Mixer(),
+        noise_module: NoiseModule = NoiseModule(),
         vca: VCA = VCA(),
     ):
-        super().__init__(modules=[pitch_adsr, amp_adsr, vco, vca])
+        super().__init__(
+            modules=[
+                pitch_adsr,
+                amp_adsr,
+                vco_1,
+                vco_2,
+                noise_module,
+                vca
+            ]
+        )
         assert sustain_duration >= 0
 
         # The convention for triggering a note event is that it has
         # the same sustain_duration for both ADSRs.
         self.pitch_envelope = pitch_adsr(sustain_duration)
```

Comment:
This is an old bug I just spotted that I'm fixing. We shouldn't be doing computations in __init__ because the can change when the parameters change. __init__ should only set parameters.